assets/src/js/frontend/give-stripe-sepa.js

```diff
@@ -19,11 +19,10 @@ document.addEventListener( 'DOMContentLoaded', function( e ) {
 	const fontStyles = [];
 	const preferredLocale = give_stripe_vars.preferred_locale;
 	const formWraps = document.querySelectorAll( '.give-form-wrap' );
-	const fontIterator = Object.entries( give_stripe_vars.element_font_styles );
```

Comment:
@mehul0810 it is a necessary change: https://github.com/impress-org/givewp/commit/e059ab1fe5175e30a3cc262decef3414e756a37f